lib/pages/home_page_v2/widgets/adzan_card/adzan_card_state_notifier.dart

```diff
@@ -0,0 +1,57 @@
+import 'package:flutter_riverpod/flutter_riverpod.dart';
+import 'package:qurantafsir_flutter/pages/prayer_time_page/prayer_times_notifier.dart';
+import 'package:qurantafsir_flutter/shared/constants/prayer_times_list.dart';
+
+final adzanCardProvider = StateProvider<AdzanState>((ref) {
```

Comment:
menambahkan provider khusus widget ini untuk menentukan waktu adzan mana yang tampil. logic berupa waktu adzan terdekat

Reply:
Nice, ini misahin logic adzan card dari logic homepage. Untuk case widget2 yang bisa berdiri independen memang perlu dibikin gini nih dibanding disatuin dengan page level state/notifier. Yang agak ribet tuh klo ada shared state antara widget yang ada di halaman, tp bagiannya agak jauh.. sama misalnya card ini yg posisinya diatas.

Thank you ndras, keren!